docs/JSON-RPC.md

```diff
@@ -146,9 +163,9 @@ Results:
 | result.id | number | The channel ID. |
 | result.name | string | The musician’s name. |
 | result.skillLevel | string | The musician’s skill level (beginner, intermediate, expert, or null). |
-| result.countryId | number | The musician’s country ID (see QLocale::Country). |
+| result.country | string | The musician’s country. |
```

Comment:
https://github.com/jamulussoftware/jamulus/pull/3249#issuecomment-2030271992

I'd suggest to add the id back...

Reply:
Definitely. No problem with adding fields to the JSON, which can be used or ignored, but we should not remove existing fields that a JSON-RPC client might potentially be using already. So retain `countryId` and `instrumentId`, and add `country` and `instrument` as new fields.

Reply:
```suggestion
| result.countryId | number | The musician’s country ID (see QLocale::Country). |
| result.country | string | The musician’s country. |
```